### DISCUSSION TITLE
Automatic installation script

### DISCUSSION THREAD

**Discussion by sup3r93** [nextcloud/all-in-one#3283] [Ideas] [answered]
for AIO, wouldn't it be better to create an auto script, which based on queries creates nextcloud instance on docker?
Honestly, I find the readme confusing and difficult to install if you use a cloudflare domain with SSL, and if Apache is already installed.

**Comment by szaimen** [comment, marked as answer]
Hi, we don't have any plans to create or maintain such a script. 

However feel free to create it yourself. You might also post it here then: https://github.com/nextcloud/all-in-one/discussions/new?category=wiki
